src/main/java/com/malinatran/constants/Status.java

```diff
@@ -5,4 +5,5 @@
     public static final String OK = "200 OK";
```

Comment:
I'm not really convinced that we need a package purely for constants. Can you think of anywhere else or another package name we could put Status and Method?